### DISCUSSION TITLE
Archive or project on home

### DISCUSSION THREAD

**Discussion by maidey** [CaiJimmy/hugo-theme-stack#391] [Q&A]
how to make Archive or projects on home like this.?

![IMG_20211006_161625.jpg](https://user-images.githubusercontent.com/90121761/136175773-9e6f664f-cc52-458a-9528-be12c507be2e.jpg)

**Comment by CaiJimmy** [comment]
You can pin a post using frontmatter's `weight` parameter. (If that's what you want to achieve)

**Comment by maidey** [comment]
> You can pin a post using frontmatter's `weight` parameter. (If that's what you want to achieve)

can you give an example?

**Comment by CaiJimmy** [comment]
> weight
> used for ordering your content in lists. Lower weight gets higher precedence. So content with lower weight will come first. If set, weights should be non-zero, as 0 is interpreted as an unset weight.
> https://gohugo.io/content-management/front-matter/

Add `weight: -1` to page's frontmatter should do the trick.
